cosmotech/data_update_quest/core/database/redis/client.py

```diff
@@ -47,6 +47,27 @@ def redis_dump(file_path, host, port, password, index_list):
         for doc in result.docs:
             json_id = json.loads(doc.json)["id"]
 
-            with open(file=path / json_id, mode="w") as f:
+            with open(file=path / (json_id + ".json"), mode="w") as f:
                 f.write(doc.json)
                 LOGGER.info(f'{T("data_update_quest.core.redis_dump.dump").format(index=index):<20} :    {json_id}')
+
+
+def file_upload(file_path, host, port, password):
+    r = get_redis_client(host=host, port=port, password=password)
```

Comment:
Use more comprehensible variable names, for example `redis_client` instead of `r`